crates/tauri-plugin-pilot/js/bridge.js

```
@@ -696,7 +696,24 @@
+    // SyntaxError from e.g. `JSON.parse('x')` must propagate, not trigger
+    // the statement fallback — otherwise the script would run twice.
+    var expr;
+    try {
+      expr = new Function("return (" + script + ")");
+    } catch (e) {
+      if (!(e instanceof SyntaxError)) throw e;
+      // Fallback for statements (`const x = 1; x`, function declarations,
+      // etc.) that the expression wrapper can't parse. Indirect eval runs
+      // in global scope and returns the completion value of the last
+      // expression (#46).
+      var indirectEval = eval;
```

Biome (2.4.11)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)